src/test/kotlin/com/finn/carrental/domain/UserServiceTest.kt

```diff
@@ -0,0 +1,74 @@
+package com.finn.carrental.domain
+
+import com.finn.carrental.api.dtos.UserRequest
+import com.finn.carrental.domain.exceptions.AlreadyExistsException
+import com.finn.carrental.domain.models.User
+import com.finn.carrental.persistence.UserRepository
+import com.finn.carrental.persistence.entities.UserEntity
+import io.mockk.every
+import io.mockk.mockk
+import org.assertj.core.api.Assertions
+import org.bson.types.ObjectId
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.springframework.dao.DuplicateKeyException
+
+class UserServiceTest() {
+
+    private val userRepository: UserRepository = mockk()
+
+    @Test
+    fun `createUser() Should add A User`() {
+        // given
+        val userService = UserService(userRepository)
+        val userRequest = UserRequest("Name", "", "")
```

Comment:
While not super important, good unit tests act as documentation as well. 
For that reason, it's good to provide tests with proper fields
e.g. 
```suggestion
        val userRequest = UserRequest("Finn", "Testhoff", "test@gmail.com")
```

Then it's also easier to understand what properties are expected as arguments. When all entries are empty string (`""`), it's impossible to know that.